os/net/lwip/src/api/sockets.c

```diff
@@ -388,7 +388,12 @@ struct lwip_sock *get_socket(int s, pid_t pid)
 	}
 
 	tcb = sched_gettcb(pid);
-	DEBUGASSERT(tcb && tcb->group);
+	if (tcb == NULL) {
```

Comment:
I think that if tcb is null then system is corrupted. So it should be stop.

Reply:
@amitpurwar I don't understand why you are trying to add this kind of strange code.

Reply:
we are analyzing the issue further and update the code accordingly